README.md

```diff
@@ -20,6 +20,13 @@ was written correctly and much more.
 
 ![Etcher](https://raw.githubusercontent.com/resin-io/etcher/master/screenshot.png)
 
+Supported Operating Systems
+---------------------------
+
+- GNU/Linux
```

Comment:
Etcher should run on virtually all modern distributions. How do you we can clarify that? I don't want to list distributions because it might give the impression that we only support the ones that we list.